test/tf/packet/main.tf

```diff
@@ -0,0 +1,19 @@
+provider "packet" {}
+
+resource "packet_project" "project" {
+  name = "go-discover:packet"
+}
+
+resource "packet_device" "discover-packet" {
+  count            = 2
+  hostname         = "tf.discover"
```

Comment:
Randomness here if hostnames are unique (not sure).

Reply:
Hostnames don't have to be unique, but i will randomize it anyways.